data-prepper-api/src/test/java/org/opensearch/dataprepper/model/event/JacksonEventTest.java

```diff
@@ -860,6 +875,7 @@ void fromEvent_with_a_JacksonEvent() {
         assertThat(createdEvent, notNullValue());
         assertThat(createdEvent, not(sameInstance(originalEvent)));
         assertThat(event.getEventHandle(), is(notNullValue()));
+        assertTrue(event.getEventHandle() instanceof DefaultEventHandle);
```

Comment:
```suggestion
        assertThat(event.getEventHandle(), instanceOf(DefaultEventHandle.class));
```

This will give a better error message if the type is incorrect. It will include the actual type.